### PR TITLE
ps-5.7-MYR-59 : 5.7 mtr - fix and re-record tests for deprecated EXPLAIN

### PR DESCRIPTION
**Pull request by george-lorch**
EXTENDED, new EXPLAIN result format, and format=json
- Removed deprecated extended keyword from all explain extended
- Relocated replace_column ids to account for new explain output structure
- Captured new Warnings in test results as part of explain outputin test results
- Captured new explain result format

### CODE REVIEW

File: mysql-test/suite/rocksdb/r/type_blob_indexes.result [hunk 1]
```diff
@@ -24,15 +24,19 @@ INSERT INTO t1 (b,t,m,l) VALUES
 (REPEAT('b',128),REPEAT('f',128),REPEAT('e',128),REPEAT('d',128)),
 (REPEAT('c',128),REPEAT('b',128),REPEAT('c',128),REPEAT('e',128));
 EXPLAIN SELECT SUBSTRING(b,16) AS f FROM t1 WHERE b IN ('test1','test2') ORDER BY f;
-id	select_type	table	type	possible_keys	key	key_len	ref	rows	Extra
-#	#	#	#	#	PRIMARY	#	#	#	#
+id	select_type	table	partitions	type	possible_keys	key	key_len	ref	rows	filtered	Extra
+#	#	#	#	#	PRIMARY	#	#	#	#	#	Using where; Using filesort
```
Comment (laurynas-biveinis) [inline mysql-test/suite/rocksdb/r/type_blob_indexes.result]: "Extra" column is not masked anymore - missing one more replace_column?

File: mysql-test/suite/rocksdb/r/type_blob_indexes.result [hunk 2]
```diff
@@ -24,15 +24,19 @@ INSERT INTO t1 (b,t,m,l) VALUES
...
+Warnings:
+#	#	#
 SELECT SUBSTRING(b,16) AS f FROM t1 WHERE b IN ('test1','test2') ORDER BY f;
 f
 
 
 EXPLAIN SELECT SUBSTRING(b,16) AS f FROM t1 USE INDEX () WHERE b IN ('test1','test2') ORDER BY f;
-id	select_type	table	type	possible_keys	key	key_len	ref	rows	Extra
-#	#	#	#	#	NULL	#	#	#	#
+id	select_type	table	partitions	type	possible_keys	key	key_len	ref	rows	filtered	Extra
```
Comment (laurynas-biveinis) [inline mysql-test/suite/rocksdb/r/type_blob_indexes.result:36]: Likewise

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

File: mysql-test/suite/rocksdb/r/type_blob_indexes.result (hunk 1 above)
Comment (george-lorch) [reply mysql-test/suite/rocksdb/r/type_blob_indexes.result]: Ohh crap, good catch. I should have seen that as I actually did fix and re-record but must have not copied back to source tree or git add-ed.

**Comment by george-lorch** [comment]
OK, fixed them, going ahead and merging.

**Pull request merged**